docs/docs/behaviors/key-press.md

```diff
@@ -40,12 +40,12 @@ For instructions to resolve such an error, click [here](../troubleshooting###Imp
 
 ## Keypad Key Press
```

Comment:
This header should be updated too. Now that we don't have consumer stuff separate, I'd argue for just "Key Press" here.

Reply:
Good call, removed from this PR.  See #413.